### PR TITLE
Update README, and probably get ready for C艹

### PR DESCRIPTION
**Pull request by redstonee**
65LJU 0O21QJL@LNGBWR}9A

### CODE REVIEW

File: CMakeLists.txt
Comment (redstonee) [inline CMakeLists.txt:1]: If I don't set(CMAKE_SYSTEM_NAME  Generic), it seems trying to build a win32 test program with the arm toolchain on Windows.

**Comment by YuzukiTsuru** [comment]
好好写commit

**Comment by YuzukiTsuru** [comment]
另外暂不考虑Windows编译，作为预编译库请使用 cmake -DCMAKE_BOARD_FILE=../lib/generic_sun8iw21.cmake ..

**Comment by chainsx** [comment]
> 65LJU 0O21QJL@LNGBWR}9A

把你吊起来玩

**Comment by redstonee** [comment]
> > 65LJU 0O21QJL@LNGBWR}9A
> 
> 把你吊起来玩

QQ图片20240104093557

**Pull request closed without merging**